loader.lua

```diff
@@ -843,6 +936,20 @@ end
 
 ------ UI
 local function drawUI()
+    local _deckURL = ""
```

Comment:
I'm a bit confused by this, it looks like `_deckURL` and `_cardBackURL` aren't used below? Maybe you forgot to rename to `valCardBackURL`? I like that you're predeclaring those as local so that they don't leak out to global scope.

Reply:
Also this would be where we would save `cardBackURL` to the `customCardBack` global variable I mentioned earlier, so that it doesn't get wiped out when the advanced panel is closed.

Reply:
Yep, the original intention was to keep them local and only retain during the re-draw but since the cardback menu disappears  I had to switch to global. these are removed now.